spec/features/managing_instrument_offline_spec.rb

```diff
@@ -0,0 +1,79 @@
+require "rails_helper"
+
+RSpec.describe "Managing Instrument Offline Reservations" do
+
+  before { login_as administrator }
+
+  let(:administrator) { FactoryGirl.create(:user, :administrator) }
+  let(:facility) { instrument.facility }
+  let(:instrument) { FactoryGirl.create(:setup_instrument) }
+
+  describe "marking an instrument offline" do
+    before(:each) do
+      visit facility_instrument_schedule_path(facility, instrument)
+      click_link "Mark Instrument Offline"
+      fill_in "offline_reservation[admin_note]", with: "The instrument is broken"
+      click_button "Take instrument offline"
+    end
+
+    it "marks the instrument offline" do
+      expect(page).to have_content("The instrument has been marked offline")
```

Comment:
What get me is the page content test would fail if `instrument.online?` was `true`. So it's timing or maybe threading? If I put a `sleep 1` before the `instrument.reload.online?` test and it that worked on CI. But I never got it to fail outside CI.